DocNet.Core/Markup/Views/layout.html

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<html>
+
+<head>
+<meta charset="utf-8">
+	<link href="../css/bootstrap.min.css" rel="stylesheet">
+	<link href="../css/bootstrap-theme.min.css" rel="stylesheet">
+	<link href="../css/custom.css" rel="stylesheet">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+	<script src="../js/bootstrap.js"></script>
+	<script src="../js/bootstrap.min.js"></script>
+	<script src="../js/collapse.js"></script>
+	<script src="../js/custom.js"></script>
+</head>
+
+<body>
+	<div class="container-fluid main">
+		<div class="page-header">
+			<h1>Header</h1>  		
+		</div>
+		<div class="row main">
+			<div class="col-md-2" id="sidebar">
+				<h2>Sidebar</h2>
+				<h3>
+					<div id="menu"><span class="glyphicon glyphicon-menu-right" aria-hidden="true"></span></div>
+					<span id="assembly" data-toggle="collapse" data-target=".list" onclick="hide(this)">Item:</span>
+				</h3>
+				<div class="collapse list">
+					<ul>
+						<li><a href="#">Item A</a></li>
+						<li><a href="#">Item B</a></li>		
+					</ul>
+				</div>
+			</div>
+			<div class="col-md-10">
+				<h2>Content</h2>
+			</div>
+		</div>
+	</div>
+	<footer class="footer">
+		<div class="container">
+			<p class="text-muted">DocNet was created by Stephen Bos, Scott Byrne, Chris Carr, and Keith Rollans.</p>
```

Comment:
For static strings similar to this (stuff that's not purely view related), it's best to put them in an external resource file (.resx) and insert them via Razor.